BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -5704,7 +5704,7 @@ public override string GetInfo()
                             output.AppendLine($"Cannister Round");
                             output.AppendLine($" - Submunition count: {binfo.subProjectileCount}");
                         }
-                        output.AppendLine($"Estimated Penetration: {ProjectileUtils.CalculatePenetration(binfo.caliber, binfo.bulletVelocity, binfo.bulletMass, binfo.apBulletMod, 940, 8.45001135e-07f, 0.656060636f, 1.20190930f, 1.77791929f):F2} mm");
+                        output.AppendLine($"Estimated Penetration: {ProjectileUtils.CalculatePenetration(binfo.caliber, binfo.bulletVelocity, binfo.bulletMass, binfo.apBulletMod, 940, 0.00000094776185184f, 0.656060636f, 1.20190930f, 1.77791929f):F2} mm");
```

Comment:
I see in the code that these same numbers are passed to `ProjectileUtils.CalculatePenetration` from a variety of places. Instead of specifying them in all the function calls, it would probably be better to set them as defaults in the `ProjectileUtils.CalculatePenetration` function so that there's a single place to change them if you need to update them later on.

Reply:
Done